### PR TITLE
ENCD-4687-add-adapter-files-to-library

### PR DESCRIPTION
**Pull request by yunhailuo**

### CODE REVIEW

File: src/encoded/tests/data/inserts/file.json [hunk 1]
```diff
@@ -2872,5 +2872,65 @@
...
+    {
+        "_test": "microRNA-seq 5' adapter",
+        "accession": "ENCFF379BCO",
+        "award": "/awards/U54HG006998/",
+        "lab": "/labs/ali-mortazavi/",
+        "file_size": 52,
+        "content_md5sum": "2b5f8ece618483ed43edd96eecacc8e4",
+        "file_format": "txt",
+        "output_type": "sequence adapters",
+        "submitted_file_name": "/Users/zach/tickets/WRAN-1139/files/full_sized_reference_files_adapters_five_prime_adapter_set4.fasta",
```
Comment (keenangraham) [inline src/encoded/tests/data/inserts/file.json]: Can probably get rid of the `/Users/zach` part.

Comment (yunhailuo) [reply src/encoded/tests/data/inserts/file.json]: Glad you mentioned that. Was debating. Will remove it. Thank you.

File: src/encoded/schemas/changelogs/reference.md [hunk 2]
```diff
@@ -4,6 +4,7 @@
 
 * *functional elements* is added to the enums list of *reference_type* property
 * Added *MouseDevSeries* enum to *internal_tags*
+* *reference_type* was updated to include the enum *adapters*
```
Comment (gabdank) [inline src/encoded/schemas/changelogs/reference.md]: it should be "sequence adapters" not "adapters"

File: src/encoded/schemas/changelogs/file.md [hunk 3]
```diff
@@ -3,6 +3,7 @@
 ### Minor changes since schema version 15
 
 * *output_type* list was updated to include the enum *elements reference*
+* *sequence adapters* can be *fasta* file
```
Comment (gabdank) [inline src/encoded/schemas/changelogs/file.md]: I am confused by this changelog update, as File schema didn't change, did it?

**Comment by yunhailuo** [comment]
> two comments to address

Made more changes and forgot the changelog. Really bad. Fixed now

File: src/encoded/schemas/library.json [hunk 4]
```diff
@@ -327,6 +327,34 @@
                     "rRNA"
                 ]
             }
+        },
+        "adapters": {
+            "type": "array",
+            "title": "Sequencing adapters",
```
Comment (keenangraham) [inline src/encoded/schemas/library.json]: Sequencing or sequence?

File: src/encoded/schemas/library.json [hunk 5]
```diff
@@ -327,6 +327,36 @@
...
+            "description": "Adapter(s) used during library preparation",
+            "uniqueItems": true,
+            "minItems": 1,
+            "items": {
+                "title": "Sequencing adapter",
+                "type": "object",
+                "additionalProperties": false,
+                "required": ["file"],
```
Comment (keenangraham) [inline src/encoded/schemas/library.json]: You could probably test this by posting library to testapp.

**Pull request merged**